src/renderer/components/reader/Reader.tsx

```diff
@@ -717,6 +719,7 @@ const mapStateToProps = (state: RootState, __: any) => {
     return {
         reader: state.reader.reader,
         mode: state.reader.mode,
+        infoOpen: state.dialog.open,
```

Comment:
If I am not mistaken, `state.dialog.open` is from the global Redux `store.getState()`, which means it will be true for any open dialog, not just the "info" dialog.
So, here is a suggestion (please review): we need a new entry like `state.reader.infoDialog`, which we can set to true in `displayPublicationInfo` of `mapDispatchToProps` (where we already `setState` on `shortcutEnable`). Then, we need to reset `state.reader.infoDialog` to false in the event listener for `Thorium:DialogClose` (where we already also reset `setState` on `shortcutEnable`). What do you think?

Reply:
Ah, even simpler:
`infoOpen: state.dialog.open && state.dialog.type === DialogType.PublicationInfoReader`

As you can see, you can test for the specific type of dialog currently opened in the browser window, which is the current local context for the global Redux `store.getState()` (therefore `state.dialog.open` is already specific to the partiular instance of the reader view)